include/snowflake/version.h

```diff
@@ -5,6 +5,6 @@
 #ifndef SNOWFLAKE_CLIENT_VERSION_H
 #define SNOWFLAKE_CLIENT_VERSION_H
 
-#define SF_API_VERSION "0.3.16"
+#define SF_API_VERSION "0.3.17"
```

Comment:
I already bumped up the version in my last commit in preparation for the next release (which is 0.3.16), so this is unneeded

Reply:
removed